verification/src/tests/transaction_verifier.rs

```diff
@@ -372,12 +390,59 @@ fn test_invalid_since_verify() {
     );
 }
 
+#[test]
+fn test_invalid_zero_length_since() {
+    // use remain flags
+    let tx = create_tx_with_lock(0xa000_0000_0000_0000);
+    let rtx = create_resolve_tx_with_transaction_info(
+        &tx,
+        MockMedianTime::get_transaction_info(1, EpochNumberWithFraction::new(0, 0, 10), 1),
+    );
+
+    let median_time_context = MockMedianTime::new(vec![0; 11]);
+    assert!(verify_since(&rtx, &median_time_context, 5, 1).is_ok(),);
```

Comment:
Is it a valid or invalid test since? For the test name `test_invalid_zero_length_since`, it's an invalid since, but `verify_since` is `is_ok()`.